Makefile

```diff
@@ -385,7 +385,10 @@ coverage: # Code coverage.
 manifests: 
 	cd crd && make manifests && make generate
 
-HELM_IMAGE_TAG ?= v0.0.2
+# Fetch the latest tag from the GitHub
+LATEST_TAG := $(shell curl -s https://api.github.com/repos/microsoft/retina/tags | jq -r '.[0].name')
```

Comment:
Are these sensibly ordered chronologically? Would doing this with Releases provide a more predictable output?

Reply:
True, updated it to use releases instead of tags